### ISSUE TITLE
item header (and the sheet templates) don't refer to the localized items (Translation issues)

### ISSUE THREAD

**Issue by TheHerugrim** [greedyj4ck/DEGENESIS-FoundryVTT#54]
https://github.com/moo-man/DEGENESIS-FoundryVTT/blob/e2989d96173322254fee9ad5d9daf609f6025d03/templates/item/item-header.html#L12

same in the other header.html files, this leads to the item headers not changing to the respective localized term
for example, the Mod sheet has MOD at the center top, but should display what is referenced in the lang file, e.g. ITEM.TypeMod
not sure where the references are made, i suspect this part to be relevant
https://github.com/moo-man/DEGENESIS-FoundryVTT/blob/e2989d96173322254fee9ad5d9daf609f6025d03/template.json#L345

the sheets like the mod-sheet also don't link to the lines from the lang files
for example, this line (https://github.com/moo-man/DEGENESIS-FoundryVTT/blob/e2989d96173322254fee9ad5d9daf609f6025d03/templates/item/item-mod-sheet.html#L18) should reference DGNS.SlotCost

![Screenshot 2022-08-06 002519](https://user-images.githubusercontent.com/110698994/183219374-a8950d91-665d-4d53-9f93-d39c9bc90c99.png)

**Comment by greedyj4ck** [comment]
We cannot translate types declared in template.json directly. Structure needs to be changed a little bit by adding additional 'label' property. Gonna add it to my 0.5.0 release to-do list :)

**Comment by greedyj4ck** [comment]
Oh wait, maybe i missed something :3

**Comment by greedyj4ck** [comment]
Figured out the way to fix it without rewriting templates. It is gonna use default structure and language entries that already exist. 

![image](https://user-images.githubusercontent.com/33233389/183249144-037a6b6e-cf75-4919-b768-ad1af7fea633.png)

I created handlebar helper to capitalize strings, and used localize with concat helper to reference proper language file entry. 

```js
 <div class="middle-name">{{localize (lookup (concat "ITEM.Type" (capitalize item.type)) 'string')}}</div>
```
Gonna use this approach for other missing translation stuff.
